audio/param.rs

```diff
@@ -408,6 +415,25 @@ impl AutomationEvent {
                 *value = val + (event_start_value - val) * exp.exp() as f32;
                 true
             }
+            AutomationEvent::SetValueCurveAtTime(ref values, start, duration) => {
+                let time_diff = ((duration.0 as f32) - (start.0 as f32)) as f32;
+                let mut progress = ((((current_tick.0 as f32) - (start.0 as f32)) as f32) / time_diff) as f32;
+                if progress < 0.0 {
```

Comment:
This should never be negative, perhaps we can add a `debug_assert!()` to check that instead of silently setting it to zero?